content/en/monitors/monitor_types/log.md

```diff
@@ -10,88 +10,84 @@ further_reading:
 
-## Notifications and log samples
+**Examples**:
 
-It is possible to add up to 10 samples of logs that triggered the monitor in the notification message.
-This is available for Slack, Jira, Webhook, Microsoft Teams, and email notifications.
+* This monitor triggers if and only if there are no logs for all services:
+  {{< img src="monitors/monitor_types/log/log_monitor_below_by_service.png" alt="Below monitor split by service" responsive="true" style="width:60%;" >}}
+* This monitor triggers if there are no logs for the service `backend`:
+  {{< img src="monitors/monitor_types/log/log_monitor_below_condition.png" alt="Below monitor for backend service" responsive="true" style="width:60%;" >}}
 
-* Samples are not displayed for recovery notifications.
+### Notifications
 
- **Enabling log samples in notifications**:
-    
-  {{< img src="monitors/monitor_types/log/activate-log-monitor-sample.png" alt="Activate log samples in message" responsive="true" style="width:50%;" >}}
-    
-  **Example for a Slack notification** 
+For detailed instructions on the **Say what's happening** and **Notify your team** sections, see the [Notifications][6] page.
 
-  {{< img src="monitors/monitor_types/log/slack-log-sample.png" alt="Slack notification example" responsive="true" style="width:50%;" >}}
- 
-### Notifications for groups
+#### Log samples
 
-Notifications from monitors split by group may include the list of the top 10 of breaching values instead of 10 log samples.
+By default, when a logs monitor is triggered:
 
- **Enabling top 10 breaching values in notifications**
+* **Monitor over a log count**: Up to 10 log samples are added to the notification message.
+* **Monitor over a facet or measure**: The top 10 facet or measure values are added to the notification message.
 
-{{< img src="monitors/monitor_types/log/activate-log-multi-monitor-sample.png" alt="Activate log samples in message" responsive="true" style="width:50%;" >}}
+These are available for notifications sent to Slack, Jira, Webhook, Microsoft Teams, and Email. **Note**: Samples are not displayed for recovery notifications.
 
-**Example for a Slack notification** 
+Notifications from monitors split by group include the top 10 breaching values instead of log samples.
 
- {{< img src="monitors/monitor_types/log/slack-log-multi-sample.png" alt="Slack notification example" responsive="true" style="width:50%;" >}}
+To disable log samples, uncheck the box next at the bottom of the **Say what's happening** section. The text next to the box is based on your monitor's grouping:
 
-## No Data alerts and Below Conditions  
+* Grouped: `Include a table of the top 10 breaching values`
+* Ungrouped: `Include a sample of 10 logs in the alert notification`
```

Comment:
Should we move this with the above explanation about what you get?

Something like:

* **Monitor over a log count**: 
           - with a group by: top 10 breaching value and their corresponding count
           - ungrouped: Up to 10 log samples are added to the notification message.
* **Monitor over a facet or measure**: The top 10 facet or measure values are added to the notification message.